packages/frontend/src/hooks/useSessionStorage.js

```diff
@@ -6,14 +6,10 @@ export const useSessionStorage = (key, defaultValue, updateRate = null) => {
     // The optional input updateRate allows for periodic checking to see if the value in session storage has changed
     // to allow multiple components using the same key to be updated
     const getStoredValue = (key, defaultValue) => {
-        // getting stored value
```

Comment:
useSessionStorage used to assume a json structure, but it's not used anymore so I made it more generic